features/bootstrap/FeatureContext.php

```diff
@@ -71,8 +79,7 @@ public function prepareTestFolders()
         }
         $this->workingDir = $dir;
         $this->phpBin = $php;
-        $this->process = new Process(null);
-        $this->process->setTimeout(20);
+        $this->process = Process::fromShellCommandline('');
```

Comment:
you should remove the instantation from here IMO as you are recalling `fromShellCommandLine` whenever you need it

Reply:
Hi @Taluu, 
First of all, thank you for your review! You're right, this line is useless, so I'll remove it.